hack/build/bazel-docker.sh

```diff
@@ -38,40 +38,36 @@ EOF
-        docker volume create ${BUILDER_VOLUME}
-    else
-        docker volume create --name ${BUILDER_VOLUME}
-    fi
+if [ -z "$(${CDI_CRI} volume list | grep ${BUILDER_VOLUME})" ]; then
+    ${CDI_CRI} volume create ${BUILDER_VOLUME}
 fi
 
 # Make sure that the output directory exists
 echo "Making sure output directory exists..."
-if [ "$KUBEVIRTCI_RUNTIME" = "podman" ]; then
-    docker run -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label=disable --rm --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} mkdir -p /root/go/src/kubevirt.io/containerized-data-importer/_out
+if [ "$CDI_CRI" = "docker" ]; then
+    ${CDI_CRI} run -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label:disable --rm --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} mkdir -p /root/go/src/kubevirt.io/containerized-data-importer/_out/gocache
 else
-    docker run -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label:disable --rm --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} mkdir -p /root/go/src/kubevirt.io/containerized-data-importer/_out
+    ${CDI_CRI} run -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label=disable --rm --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} mkdir -p /root/go/src/kubevirt.io/containerized-data-importer/_out/gocache
 fi
 
 echo "Starting rsyncd"
 # Start an rsyncd instance and make sure it gets stopped after the script exits
-if [ "$KUBEVIRTCI_RUNTIME" = "podman" ]; then
-    RSYNC_CID_CDI=$(docker run -d -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label=disable --expose 873 -P --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} /usr/bin/rsync --no-detach --daemon --verbose)
+if [ "$CDI_CRI" = "docker" ]; then
+    RSYNC_CID_CDI=$(${CDI_CRI} run -d -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label:disable --expose 873 -P --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} /usr/bin/rsync --no-detach --daemon --verbose)
 else
-    RSYNC_CID_CDI=$(docker run -d -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label:disable --expose 873 -P --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} /usr/bin/rsync --no-detach --daemon --verbose)
+    RSYNC_CID_CDI=$(${CDI_CRI} run -d -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label=disable --expose 873 -P --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} /usr/bin/rsync --no-detach --daemon --verbose)
 fi
 
 function finish() {
-    docker stop ${RSYNC_CID_CDI} >/dev/null 2>&1 &
-    docker rm -f ${RSYNC_CID_CDI} >/dev/null 2>&1 &
+    ${CDI_CRI} stop ${RSYNC_CID_CDI} >/dev/null 2>&1 &
+    ${CDI_CRI} rm -f ${RSYNC_CID_CDI} >/dev/null 2>&1 &
 }
 trap finish EXIT
 
-if [ "$KUBEVIRTCI_RUNTIME" = "podman" ]; then
-    RSYNCD_PORT=$(docker port $RSYNC_CID_CDI | cut -d':' -f2)
+if [ "$CDI_CRI" = "docker" ]; then
+    RSYNCD_PORT=$(${CDI_CRI} port $RSYNC_CID_CDI 873 | cut -d':' -f2)
 else
-    RSYNCD_PORT=$(docker port $RSYNC_CID_CDI 873 | cut -d':' -f2)
+    RSYNCD_PORT=$(${CDI_CRI} port $RSYNC_CID_CDI | cut -d':' -f2)
 fi
```

Comment:
why they are different ?
both use port 873 at the end (lines 55- 59) unless for one of them it is a default or something ?

Reply:
Done!